src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { Todo } from './types/Todo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todosLegacy, setTodosLegacy] = useState<Todo[]>(todos);
+  const [newTitle, setNewTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const findLargestId = () => {
+    return Math.max(...(todosLegacy.map(todo => todo.id)));
+  };
+
+  const resetForm = () => {
+    setNewTitle('');
+    setSelectedUserId(0);
+  };
```

Comment:
Clear form errors. [You have a bug](https://www.loom.com/share/b91753b4643f41129a4586b4dd121eb6).